gillespy2/core/model.py

```diff
@@ -62,6 +62,24 @@ def export_SBML(gillespy_model, filename=None):
     return export(gillespy_model, path=filename)
 
 
+def export_StochSS(gillespy_model, filename=None, return_stochss_model=False):
```

Comment:
It may be a good idea to move this inside of the `Model` class and change the name to `to_stochss()` so it better fits the new `to_json()` and `from_json()` functions. Neither is required, just something to consider.